### ISSUE TITLE
Dash Daq slider defaults to the negative minimum value when set to value is set to 0.

### ISSUE THREAD

**Issue by mithi** [plotly/dash-daq#102]
I do something like this:
``` python
dash_daq.Slider( 
    id=name,
    min=-120,
    max=120,
    value=0,
    size=150,
    updatemode=UPDATE_MODE,
    handleLabel={"showCurrentValue": True,"label": angle},
    step=5,
  )
```
And my slider defaults to `-120` which is the default minimum value instead of `0`. When I use a `value=10` my slider defaults correctly to `-10` as expected. 

The version I'm using Dash DAQ '0.4.0'

I'm using dash in my project here: https://github.com/mithi/hexapod-robot-simulator
which I use that code snippet in this file https://github.com/mithi/hexapod-robot-simulator/blob/master/widgets/joint_input_maker.py

**Comment by alexcjohnson** [comment]
Good catch @mithi! Looks like we fail to distinguish `undefined` from `0` in several places, at least:

https://github.com/plotly/dash-daq/blob/42111db5c1c07584d4dec41cf1f462e17adc91d5/src/fragments/Slider.react.js#L69

(that's probably the most important one)

https://github.com/plotly/dash-daq/blob/42111db5c1c07584d4dec41cf1f462e17adc91d5/src/fragments/Slider.react.js#L96

(that looks like an edge case, but should also be addressed)

**Comment by mithi** [comment]
@alexcjohnson , Thanks for taking the time to read and investigate this issue!
